MEK_Installer.pyw

```diff
@@ -372,8 +420,10 @@ class MekInstaller(tk.Tk):
 
     def __init__(self, *args, **kwargs):
         tk.Tk.__init__(self, *args, **kwargs)
-        self.title("MEK installer v2.2.2")
-        self.geometry("480x400+0+0")
+        self.title("MEK installer v%s.%s.%s" % (2,3,0))
+        # Default scale == 1.333
+        self.tk.call("tk", "scaling", "1.666")
```

Comment:
If you want to universally increase the scaling by a certain amount, you don't want to explicitly set the scaling, you want to scale it. This should achieve the effect you're looking for, and should actually work on high density screens.
```python
self.tk.call('tk', 'scaling', 1.25 * self.tk.call("tk", "scaling"))
````

Reply:
My thing does work. And the default scale is always 1.333 as Python never changes it.
Also, tk calls always return strings afaik. So that code would not work without a float cast.

Reply:
The reason I typed it like that is because it was returned as a float for me(doing the cast is trivial though so w/e). Regardless, I worry the default scaling may not be 1.333 depending on the environments. I would like to see a definitive way to determine where/how the scaling is originally set. If it's used as a global UI scaling factor, I feel that tkinter should be smart enough to at least try to generate a sane default on startup, rather than a one-size-sorta-fits-all approach.

Reply:
I checked the Tkinter source, and it does not at all have the word scaling in it. Tk documentation also suggest to set your own with 1.333 always being the default.

Did another super thorough check for the word. (Unlike me just checking the source manually last time)
```
michelle@nessy:/usr/lib/python3.7/tkinter$ cat *.py | grep scaling
michelle@nessy:/usr/lib/python3.7/tkinter$ 
```
cat prints the contents of each file it gets as an argument. grep will return any lines containing the given pattern.
This means there is no mention of it.

yeah, it's dumb.

Reply:
I just checked the tk8.6.10 source code.
There is the scaling cmd, and other than that every mention of "scaling" in all of the mentions are in comments and documentation.

Scaling is meant to be set up by the user themselves >.>

Reply:
I appreciate the effort. Yeah, that is really dumb and weird. It also seems there isn't a good universal way to determine the screen's DPI to help calculate a better scaling factor. All I could find is that a scaling factor of 1.0 is for 72DPI. This means 1.333 is 96DPI, and 1.666 is 120DPI.